api-specificatie/Bevraging-Ingeschreven-Persoon/openapi.yaml

```diff
@@ -780,8 +895,18 @@ paths:
         'default':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/master/api-specificatie/common.yaml#/components/responses/default"
       tags: 
-      - reisdocumenten
+      - Historie
 components:
+  parameters:
+    burgerservicenummer:
```

Comment:
we gebruiken burgerservicenummer op 3 manieren:
- als path parameter
- als query parameter
- als response object property

voor nu niet blocking, maar wellicht kunnen we een basiscomponent burgerservicenummer definiëren, en die met kleine aanpassingen hergebruiken voor de drie varianten?